Maple2.File.Parser/Xml/AI/Node.cs

```diff
@@ -4,87 +4,89 @@
 using System.Numerics;
 using M2dXmlGenerator;
 using Maple2.File.Parser.Enum;
+using System.Security.Cryptography.X509Certificates;
 
 namespace Maple2.File.Parser.Xml.AI;
 
+public class Entry {
+    public string name = string.Empty;
+}
 
-public partial class Node {
-    [XmlAttribute] public string name = string.Empty;
+public partial class Node : Entry {
```

Comment:
given you are no longer using the generated parsing structure, why not just split the hierarchy here

Reply:
The current splitting method already works & I don't wanna introduce more work than necessary. It'd be a fairly big refactor for an end result that isn't any different once it gets ingested & added to the db

Reply:
not a fan of making things worse because it's easier

Reply:
Issue is that now that its established code making that kind of change is a risk factor if it isn't necessary. It'd be a whole lot of class duplication for something that'd be immediately thrown out & put into records.

From when Zin & I were talking last night it also seems like the feature node will need to be resolved in Maple2.File.Ingest with FeatureLocaleFilter. Unless that's also in Maple2.File the whole script will probably need to be cloned just to strip out or disable the feature nodes that shouldn't be enabled.

The biggest concern I have is that this whole parser will need to be rewritten significantly again just to support the early splitting because it relies on the specific node types that need to be created being known at construction time, something that'll balloon the parser with the expanded out node types.

Reply:
From a philosophical standpoint, think of these as two independent projects. We should not be doing special coordination and duplication is the expected pattern. It should generally be viable for a third party to make use of this one without needing special coordination.

Reply:
You do have another option though, which is to remove all this code from `Maple2.File` then you can hide whatever implementation you want directly in Ingest.

Reply:
Yeah I'll prob go with the second option. I'm not keen on implementing the other half of a fully featured XML parser lol. For this PR should I cancel it & just copy over what I've done to Ingest instead?

Reply:
Yea just close this PR. When you move it over, you'll want to drop the `Maple2.File` code (so you're not doing some unnecessary double conversion XML -> Maple2.File -> Maple2.Model)